src/settings_gui.cpp

```diff
@@ -215,6 +235,14 @@ struct GameOptionsWindow : Window {
 				}
 				break;
 
+			case WID_GO_REFRESH_RATE_DROPDOWN: // Setup refresh rate dropdown
+				for (uint i = 0; i < _refresh_rates.size(); i++) {
+					auto item = new DropDownListParamStringItem(STR_GAME_OPTIONS_REFRESH_RATE_ITEM, i, false);
+					item->SetParam(0, *std::next(_refresh_rates.begin(), i));
+					list.emplace_back(item);
```

Comment:
shouldn't be emplace_back here - the object has already been constructed. Use push_back instead

Reply:
Blame @TrueBrain for that one!